Source/DafnyLanguageServer.Test/Synchronization/TextReplacementTest.cs

```diff
@@ -1,110 +1,90 @@
-using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
 using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.Workspace;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Synchronization {
-  public class TextReplacementTest : SynchronizationTestBase {
+  public class TextReplacementTest {
     [Fact]
-    public async Task ReplaceSingleLineTextAtStart() {
+    public void ReplaceSingleLineTextAtStart() {
```

Comment:
If I understand correctly ,you completely remove the dependency of this test to the language server. I feel very suspicious about that as these are the language server tests, the transmission protocol, not tests about Omnisharp. Please justify.

Reply:
The previous test was pretending to be a high level using LSP APIs, but was actually mixing them with C# APIs to access internal state. I don't think it made sense. Anyways, the internal APIs it was using don't exist anymore, and now it's using more stable TextBuffer C# APIs.

Reply:
Ok, so actually these tests are not testing anything of our own code we wrote, so if they are not useful anymore, what about we remove them? It looks like we are testing an existing library. Unless I miss something.

Reply:
TextBuffer is our own code.